### PR TITLE
Add required_ruby_version to gemspec

### PR DESCRIPTION
**Pull request by dogenpunk**
This change leads up to a fix for hanami/hanami#924

### CODE REVIEW

File: hanami-cli.gemspec [hunk 1]
```diff
@@ -27,6 +27,8 @@ Gem::Specification.new do |spec|
     f.match(%r{^(test|spec|features)/})
   end
 
+  spec.required_ruby_version = ">= 2.4.0"
```
Comment (davydovanton) [inline hanami-cli.gemspec]: hanami cli works with jruby too. How we can handle it here?

Comment (dogenpunk) [reply hanami-cli.gemspec]: That's a good question. I'm trying to get up to speed on jruby support. The other hanami gems seem to use this setting already so I thought it was an oversight here. I'll come back to this after I've completed this process for them.

Comment (AlfonsoUceda) [reply hanami-cli.gemspec]: Folks, this is related to jruby 9.1 because it targets ruby 2.3, jruby 9.2 will target ruby 2.5

**Comment by davydovanton** [comment]
agree with @AlfonsoUceda 👏

**Comment by AlfonsoUceda** [comment]
can you update to jruby 9.2.0.0? it was release some days ago. thanks!

**Comment by AlfonsoUceda** [comment]
@dogenpunk can you update travis ci file to reference jruby 9.2.0.0? thanks!

**Comment by IvanShamatov** [comment]
Closing this one as outdated. The initial problem was solved in hanami

**Pull request closed without merging**